tests/drbg.cc

```diff
@@ -283,8 +313,132 @@ TEST_P(DrbgPRTrueSuite, KAT)
   // TODO: Does it provide predictive resistance?
 }
 
+// ----- EverCrypt -------------------------------------------------------------
+
+typedef EverCryptSuite<CAVPTestCase> DrbgNRSuiteEverCrypt;
+
+TEST_P(DrbgNRSuiteEverCrypt, KAT)
+{
+  EverCryptConfig config;
+  CAVPTestCase test;
+  tie(config, test) = this->GetParam();
+
+  bool skip = false;
+  Spec_Hash_Definitions_hash_alg hash;
+  cavp_to_hash(test.hash, skip, hash);
+  if (skip) {
+    cout << "Skipping \"" << test.hash << "\"" << endl;
+    return;
+  }
+
+  // Init
+  EverCrypt_DRBG_state_s* state = EverCrypt_DRBG_create(hash);
+  bool res = EverCrypt_DRBG_instantiate(state,
+                                        test.PersonalizationString.data(),
+                                        test.PersonalizationString.size());
+  ASSERT_TRUE(res);
+
+  // FIXME: EntropyInput?
```

Comment:
It looks like evercrypt doesn't allow you to feed this in. so we can't use these KATs because they have an `EntropyInput`.

Reply:
Hm... any idea how `EverCrypt_DRBG_*` is supposed to be used then? There must certainly be some way to feed the DRBG some initial entropy before calling `EverCrypt_DRBG_generate` ... There is `EverCrypt_DRBG_reseed`. But... that seems kinda weird?

Reply:
Not weird. It's reading system randomness as it's supposed to. But it doesn't have APIs for testing.

Reply:
Can you elaborate? I meant that it would be kinda weird when "reseed" is supposed to be called directly after "initiate". Doesn't it?

Reply:
I don't understand the question. There's no need to call `reseed`.

Reply:
Ahh... Now I understand your point. It is reading system randomness in `EverCrypt_DRBG_instantiate(...)`. I thought that Hacl (and EverCrypt?) don't do that.

Reply:
Right, verified code doesn't :)